src/client/driver/command-executors/execute-action-command.js

```diff
@@ -2,76 +2,107 @@ import hammerhead from '../deps/hammerhead';
 import testCafeCore from '../deps/testcafe-core';
 import testCafeRunner from '../deps/testcafe-runner';
 import testCafeUI from '../deps/testcafe-ui';
-import { ActionElementNotFoundError, ActionElementIsInvisibleError } from '../../../errors/test-run';
+import {
+    ActionElementNotFoundError,
+    ActionElementIsInvisibleError,
+    DragSourceNotFoundError,
+    DragSourceIsInvisibleError,
```

Comment:
We shouldn't use term `sourceSelector`. We have just `selector` which is first argument of the action. Use it.